diff --git a/src/derive.cc b/src/derive.cc
index ef7105c..25c3134 100644
--- a/src/derive.cc
+++ b/src/derive.cc
@@ -738,7 +738,7 @@ namespace giac {
   }
   static string texprintasderive(const gen & feuille,const char * sommetstr,GIAC_CONTEXT){
     if (feuille.type!=_VECT)
-      return gen2tex(feuille,contextptr)+"'";
+      return "\\left("+gen2tex(feuille,contextptr)+"\\right)'";
     return "\\frac{\\partial \\left("+gen2tex(feuille._VECTptr->front(),contextptr)+"\\right)}{\\partial "+gen2tex(feuille._VECTptr->back(),contextptr)+"}";
   }
   static define_unary_function_eval4_quoted (__derive,&step_derive,_derive_s,printasderive,texprintasderive);
